homeassistant/components/plugwise/strings.json

```diff
@@ -16,6 +16,7 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "invalid_setup": "Add your Adam instead of your Anna, see the Home Assistant Plugwise integration documentation for more information",
+      "retry": "Something went wrong, please retry",
```

Comment:
This is an odd message/thing to do? Why should a user retry? Instead, we should retry ourselves in code directly.

Reply:
Tnx for the suggestion, will look at this tonight.